test/tools/router_check/router.cc

```diff
@@ -65,7 +61,6 @@ bool RouterCheckTool::compareEntriesInJson(const std::string& expected_route_jso
   Json::ObjectSharedPtr loader = Json::Factory::loadFromFile(expected_route_json);
```

Comment:
can we use yaml to load this instead? if you want to put a TODO so you don't have to make a proto, etc that's fine too.